libs/librepcb/librarymanager/librarylistwidgetitem.cpp

```diff
@@ -55,21 +55,16 @@ LibraryListWidgetItem::LibraryListWidgetItem(workspace::Workspace& ws,
     if (!icon.isNull()) {
       mUi->lblIcon->setPixmap(icon);
     }
-    if (isRemoteLibrary()) {
-      mUi->lblLibraryType->setText(tr("(remote)"));
-      mUi->lblLibraryType->setStyleSheet("QLabel { color: red; }");
-    } else {
-      mUi->lblLibraryType->setText(tr("(local)"));
-      mUi->lblLibraryType->setStyleSheet("QLabel { color: blue; }");
-    }
     mUi->lblLibraryName->setText(name);
     mUi->lblLibraryDescription->setText(description);
-    mUi->lblLibraryUrl->setText(libDir.toRelative(ws.getLibrariesPath()));
+    QString path = libDir.toRelative(ws.getLibrariesPath());
+    path.replace("local/", "<font color=\"blue\">local</font>/");
+    path.replace("remote/", "<font color=\"red\">remote</font>/");
```

Comment:
Inline styles like it's 1998 :smile: 

(Yes, I know that this is fine.)

Reply:
Yep, thanks to that I could remove another QLabel to make the UI more clean :wink: